fedgov-cv-message-builder/src/test/java/gov/usdot/cv/msg/builder/IntersectionSituationDataBuilderTest.java

```diff
@@ -135,21 +135,21 @@ public void mapMessageWithNodeSpeedLimitTest() {
     @Test
     public void rgaBaseLayerMessageWithFramePlusRGATypeTest() {
         String filePath = "src/test/resources/samplerga.json";
-        String expectedHexString = "00000a66d9b231ea6b5f708e13f410401808101807f7e820b420020690";
+        String expectedHexString = "00000a66d9b231ea6b5f708e13f4105008a80c0602040601fdfa082d080081a400";
         runIntersectionSituationDataBuilderTests(filePath, expectedHexString);
     }
 
```

Comment:
Add more unit test cases to test the Relative Road Authority ID and Mapped GeomID. Make sure to test a node greater than 127 for Relative Road Authority ID.